### DISCUSSION TITLE
Embedding the latest 4 jobs on our website

### DISCUSSION THREAD

**Discussion by ayrancd** [bullhorn/career-portal#418] [Q&A] [answered]
I'd like to show the latest 4 job positions on my website homepage https://dbistaffing.com/ is there any JS code or API I can use to do that? Thank you!

**Comment by inTulsa** [comment]
Hi, I was able to achieve this by creating a RSS feed to post "featured" jobs to my website. This method should work, in your case, to get the latest 4 jobs. Not sure where your site is hosted but you may be able to use something like Zapier to pull the RSS feed and load the data to your homepage.

I don't work for Bullhorn, just stumbled upon this post and thought I'd share.

**Reply by ayrancd** [reply]
Hi,

How did you get the RSS url? This is our Job portal https://jobs.dbistaffing.com

**Reply by inTulsa** [reply]
The RSS feed is only available through the "Advanced Hosting" feature of the Bullhorn Career Portal. It looks like your site is using the basic career portal which only hosts static files.

You can follow this thread #412 to easily set up advanced hosting with Heroku which worked great for me. Once that is up and running, you'll be able to gain access and create RSS feeds.

**Comment by ayrancd** [comment, marked as answer]
Just found a solution:

`<?php
    $ch = curl_init();
    curl_setopt($ch, CURLOPT_URL, "https://public-rest30.bullhornstaffing.com/rest-services/1VOIP/query/JobBoardPost?where=(isOpen=true)&fields=id,title,publishedCategory(id,name),address(city,state),employmentType,dateLastPublished,publicDescription,isOpen,isPublic,isDeleted&count=4&orderBy=-dateLastPublished&start=0");

    curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
    $output = curl_exec($ch);

    curl_close($ch);
    $ch = curl_init();

    $out = json_decode($output, true);
    for($i=0; $i<=count($out); $i++){
        $desc = strip_tags($out["data"][$i]["publicDescription"]);
        $description = strlen($desc) > 100 ? substr($desc,0,100)."..." : $desc;?>
        <div class="col6">
            <div class="jobs-wrapper">
                <a class="job-item" href="https://jobs.dbistaffing.com/#/jobs/<?=$out["data"][$i]["id"]?>">
                    <span class="job-title-category">
                        <span class="job-title"><?=$out["data"][$i]["title"]?></span> 
                        <span class="job-category"><?=$out["data"][$i]["publishedCategory"]["name"]?></span>
                    </span>
                    <span class="job-location-category">
                        <span class="job-location"><?=$out["data"][$i]["address"]["city"]?><span class="ng-scope">,&nbsp;</span><?=$out["data"][$i]["address"]["state"]?></span>
                        <span class="job-separator">|</span>
                        <span class="job-type"><?=$out["data"][$i]["employmentType"]?></span> 
                        <span class="job-date">Added - 07/30/21</span>
                    </span>
                    <span class="job-description"><?=$description?></span> 
                </a>
            </div>
        </div>
<? }?>`

**Reply by inTulsa** [reply]
That works! Nicely done.

**Reply by pbearne** [reply]
FYI. The [Matador Jobs](https://matadorjobs.com/) Plugin has a shortcode that lists jobs (and more) out of the box we can do this because we import the Job into WP as a CPT. This is a huge boost for SEO and branding as you can apply the theme branding to the jobs board.
